docs/script.rst

```diff
@@ -274,14 +271,13 @@ Prints::
     Mark=Williams
     Sudha=Kumari
 
+A closure has two other important features. First, it can access variables in the scope where it is defined,
```

Comment:
I didn't make a change related to this point, but: I was a little confused reading the documentation as to whether Groovy uses the term `closure` in a more general way than I'm used to. For the examples here, like what is passed into `Map.each()`, I would normally just use the term _function_, rather than _closure_. I would only use the term _closure_ when the first feature here -- accessing in-scope variables -- is in play. The second feature is something I'd normally attribute to an _anonymous function_, whether or not that function was also a closure. But I left all of this alone.